test/e2e/scale_test.go

```
@@ -103,7 +104,7 @@ func testScaleToWithin(t *testing.T, logger *logging.BaseLogger, scale int, dura
 			svc, err := test.CreateLatestService(logger, clients, names, options, fopt...)
 			if err != nil {
 				t.Errorf("CreateLatestService() = %v", err)
-				return
+				return nil
```

why not return the error?